src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final int FIGURE_COUNT = 3;
+
     public String getRandomColor() {
-        return null;
+        return Color.values()[new Random().nextInt(FIGURE_COUNT)].name();
```

Comment:
Think about which variables should be local in the method and which should be class-level

see common mistakes https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery